Documentation/helm-operator.md

```diff
@@ -99,14 +66,19 @@ To deploy from a local build from your development environment:
 cd cluster/charts/rook-ceph
 helm install --namespace rook-ceph rook-ceph .
 ```
-For Helm `v2.x` the `--name` flag must be specified instead of just `rook-ceph`, e.g., `--name rook-ceph`.
 
 ## Uninstalling the Chart
 
+To see the currently installed Rook chart:
+
+```console
+helm ls -n rook-ceph
+```
+
 To uninstall/delete the `rook-ceph` deployment:
 
 ```console
-helm delete --purge rook-ceph
+helm delete -n rook-ceph rook-ceph
```

Comment:
I think `-n` is short for `--namespace`. I think it's a good practice to always try to spell out env vars when possible so that the commands are more self documenting.

Reply:
Sure i'll spell it out